### ISSUE TITLE
Group permissions are not inherited

### ISSUE THREAD

**Issue by thurban** [Cacti/plugin_weathermap#3]
For a user to be able to see weathermap tab, he needs to have permission set to (Weathermap:View) at a user level, permission set in the group the user belongs to doesn't seem to be checked.
For example nFlow plugin, does inherit permissions from the group the user belongs to and show the appropiate tab (even if the user don't have that permission).

**Comment by TheWitness** [comment]
This should be resolved now @thurban.  The permission check is using the Cacti Plugin API now for checking tab/UI permissions which includes Group permissions.  Of course the Weathermap permission model remains, which will get a good looking at a some point to make those users include Cacti Groups.  Separate issue though.

This issue will be resolved once this pull request is merged: https://github.com/Cacti/plugin_weathermap/pull/39
